app/views/layouts/_nav.html.erb

```diff
@@ -28,6 +28,9 @@
           <li><%= link_to "<span class='fas fa-fw fa-code'></span> REST API Documentation".html_safe, api_v1_swagger_ui_path, target: '_blank' %></li>
 				</ul>
 			</li>
+			<li class="left-border-0-5">
```

Comment:
Small nit - it looks like the whitespace is inconsistent in this file (some places is spaces, others is tabs).  It would be nice if it were harmonized for readability.